lib/bot.js

```diff
@@ -33,6 +33,8 @@ function Bot(options) {
     channels: this.channels
   });
 
+  this.client.autosendcmd = options.autosendcmd;
```

Comment:
No need to set this on the client, just do `this.autoSendCommands = options.autoSendCommands;` instead (`autosendcmd` -> `autoSendCommands` too), and move it up to the other option-specific assignments (line 25 etc.).